tests/web3/undefinedOpcode.expected.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": true
+    },
+    {
+        "jsonrpc": "2.0",
+        "id": 2,
+        "result": true
+    },
+    {
+        "jsonrpc": "2.0",
+        "id": 3,
+        "error": {
+            "message": "VM Exception while processing transaction: EVMC_UNDEFINED_INSTRUCTION",
```

Comment:
I feel like this should be `invalid opcode` instead of `EVMC_UNDEFINED_INSTRUCTION` here.

Reply:
NVM, this is fine.